gammapy/scripts/spectrum_pipe.py

```diff
@@ -87,10 +91,22 @@ def run_fit(self):
         self.egm = SpectrumEnergyGroupMaker(stacked_obs)
         self.egm.compute_groups_fixed(self.config['fp_binning'])
 
+        try:
+            nsigma_ul_threshold = self.config['fit']['nsigma_ul_threshold']
+        except:
+            nsigma_ul_threshold = 1.5
+        try:
+            nsigma_ul = self.config['fit']['nsigma_ul']
+        except:
+            nsigma_ul = 3.
```

Comment:
This can be written much shorter with `self.config['fit'].get('nsigma_ul_threshold', 1.5)`, can you add the default values to the docs? An maybe change them to `nsigma_ul=2` and `nsigma_ul_threshold=1`, those are the values used by the Fermi catalogs and the HGPS.